### ISSUE TITLE
Compilation issues "Undefined symbol: _OBJC_CLASS_"

### ISSUE THREAD

**Issue by StevenMasini** [parse-community/Parse-SDK-iOS-OSX#1665]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-iOS-OSX/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-iOS-OSX/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse ObjC SDK](https://github.com/parse-community/Parse-SDK-iOS-OSX/releases). 

### Issue Description

Getting the following compilation errors.

* Undefined symbol: _OBJC_CLASS_$_FBSDKAccessToken
* Undefined symbol: OBJC_CLASS_$_FBSDKApplicationDelegate
* Undefined symbol: _OBJC_CLASS_$_FBSDKSettings

### Steps to reproduce

Compile a project that make use of `Parse/FacebookUtils` using any version between `1.9.1` to latest `1.9.3`

### Actual Outcome

![image](https://user-images.githubusercontent.com/1907562/193730837-9c52c406-701f-467a-951c-611c153786a9.png)

### Expected Outcome

No compilation errors.

###  Environment

XCode 14.0.0, any arm64 devices

Client
- Parse ObjC SDK version: `1.9.1`, `1.9.2`, `1.9.3`

Server
- Parse Server version: N/A
- Operating system:  N/A
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc):  N/A

Database
- System (MongoDB or Postgres):  N/A
- Database version:  N/A
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc):  N/A

### Logs
 N/A

**Comment by StevenMasini** [comment]
Seems to be a pretty old issue tho https://stackoverflow.com/questions/64294415/build-error-with-parse-ios-sdk-and-facebook/73943043#73943043

**Comment by mtrezza** [comment]
Closing as related to https://github.com/parse-community/Parse-SDK-iOS-OSX/issues/1663

**Comment by StevenMasini** [comment]
@mtrezza I will try to find some time this weekend to work on it. The one solution that truly matters nowadays is SPM. But lots of developers are still relying on Cocoapods for their projects. 

I think it's possible to add Cocoapods as a solution to setup Parse ObjC SDK.

**Comment by mtrezza** [comment]
In general, if you want to invest your time into trying to add SPM, it would be better spent I think. There are some PRs that need adaption, but they show that adding SPM works as a proof of concept.

**Comment by mathieulb** [comment]
Is this really about versions 1.9.1 thru 1.9.3, or actually 1.19.1 thru 1.19.3, which are the most recent ones ? But I have that problem with any 1.19.x, even 1.19.0... and I can't seem to install 1.18.0, for other reasons...

**Comment by danipralea** [comment]
@StevenMasini how did you fixed it? I have the same issue and it's just driving me crazy. I did everything and I can't get rid of it
![image](https://user-images.githubusercontent.com/768893/198007351-b4373cf3-1989-4bda-a361-b1dca2bbbca4.png)

**Comment by StevenMasini** [comment]
@danipralea You can point to my repo until #1666 gets merged.

Replace this line in your `Podfile`
```
pod 'Parse', :git => 'https://github.com/StevenMasini/Parse-SDK-iOS-OSX.git', :commit => '75ad3d3608cba0ecbc4142a64de871cf3f84f172'
```

It will point to this commit on my fork https://github.com/parse-community/Parse-SDK-iOS-OSX/pull/1666/commits/75ad3d3608cba0ecbc4142a64de871cf3f84f172

**Comment by mtrezza** [comment]
@StevenMasini Is #1666 ready to be merged?

**Comment by danipralea** [comment]
thank you @StevenMasini that finally fixed it! 🙏 🙏

**Comment by mtrezza** [comment]
Thanks for confirming @danipralea , I'll go ahead and merge the PR.

**Comment by mtrezza** [comment]
🎉 This change has been released in version [1.19.4](https://github.com/parse-community/Parse-SDK-iOS-OSX/releases/tag/1.19.4)
